common.yml

```diff
@@ -740,6 +742,17 @@ functions:
         set -e
         GOPATH=$PWD go get github.com/10gen/ftdc-utils/cmd/ftdc
 
+  "fetch gometalinter" :
```

Comment:
I chose to use gometalinter, which is a wrapper that can run many different linters/formatters, instead of running gofmt directly. This is because gofmt doesn't return a non-zero exit code if the files aren't correctly formatted, so figuring out if formatting failed required some extra work that didn't seem worth it.

Gometalinter is deprecated in favor of golangci-lint, but I talked to Sam Kleinman on Evergreen and he said gometalinter is fine for our purposes.

Reply:
I think "fine for our purposes" isn't the same as "a good choice for the future".  Starting with deprecated tools seems the wrong approach.

Reply:
Fair point - I'll update it to golangci-lint